csharp/Platform.Data.Doublets.Sequences/Unicode/UnicodeSymbolsListToUnicodeSequenceConverter.cs

```diff
@@ -92,6 +92,10 @@ public TLinkAddress Convert(IList<TLinkAddress>? list)
         {
             _index.Add(list);
             var sequence = _listToSequenceLinkConverter.Convert(list);
+            if (EqualityComparer<TLinkAddress>.Default.Equals(default, sequence))
+            {
+                return _unicodeSequenceMarker;
+            }
```

Comment:
We should check list, not sequence, and the check should be moved to the beginning of the method. We also do not need to index an empty list, because it is constant in our point of view.